visualization/app/codeCharta/ui/filePanel/filePanel.component.spec.ts

```diff
@@ -141,6 +144,14 @@ describe("filePanelController", () => {
 			expect(filePanelController["_viewModel"].selectedFileNames.partial).toBeNull()
 			expect(filePanelController["_viewModel"].selectedFileNames.single).toBeNull()
 		})
+
+		it("should set the pictogram colors in view model", () => {
+			filePanelController.onFileSelectionStatesChanged(fileStates)
+
+			expect(filePanelController["_viewModel"].pictogramFirstFileColor).toBe("#808080")
+			expect(filePanelController["_viewModel"].pictogramLowerColor).toBe("#ff0E0E")
+			expect(filePanelController["_viewModel"].pictogramUpperColor).toBe("#69FF40")
```

Comment:
does this test make sense? Because the whole point of getting the color via the settings service is that we could update it centrally, therefore i don't know if we should check here for a specific value that we would need to update if we change the value in the settings service...

Reply:
That's true. If you use fixed values in your test, you have to adapt them, when you change your production code. But this way, you will notice, when it actually changes.
The developer should see a failing test when changing these values and get thinking: "Should I really change these color values?" and if so, just adapt the test.